src/services/dashboards/widgets/config.ts

```diff
@@ -40,6 +38,16 @@ export const GROUP_BY = {
     ACCOUNT: 'account',
 } as const;
 
+export const CHART_TYPE = Object.freeze({
+    TREEMAP: 'TREEMAP',
+    MAP: 'MAP',
+    LINE: 'LINE',
+    STACKED_COLUMN: 'STACKED_COLUMN',
+    DONUT: 'DONUT',
+    PIE: 'PIE',
+    WAFFLE: 'WAFFLE',
+} as const);
```

Comment:
When using 'as const', you don't have to use Object.freeze!

Reply:
oh i missed it, thanks!